openvdb_vendor/package.xml

```diff
@@ -23,7 +23,9 @@
   <buildtool_depend>ament_cmake_vendor_package</buildtool_depend>
   <buildtool_depend>git</buildtool_depend>
 
+  <depend>libboost-iostreams-dev</depend>
   <depend>libboost-thread-dev</depend>
+  <depend>libgdal-dev</depend>  <!-- For libblosc-dev -->
```

Comment:
`libblosc-dev` needs to be added to the rosdistro index for this to be listed properly. But it's way past bedtime here, so that's something for tomorrow :slightly_smiling_face: